app/controllers/v1/mongo_sessions_controller.rb

```diff
@@ -0,0 +1,45 @@
+module V1
+  class MongoSessionsController < ApplicationController
+    before_action :authenticate_user!
```

Comment:
@MaciejGawel W ogóle inaczej to zrobiłem. Stamtąd wywaliłem, dla `ForamsController` ustawiłem specjalny `before_action :authenticate_user` - bez wykrzyknika, w sensie - jeśli jest token i jest poprawny, to go zaloguj, a jeśli nie, to niech ma dostęp do "wspólnych" foramów. Co o tym myślisz?